### DISCUSSION TITLE
navigraph AIRAC 2202 rev2 Missed approach inserted in flight plan

### DISCUSSION THREAD

**Discussion by kiokio0987** [flybywiresim/aircraft#6849] [General]
Navigraph has been updated to "AIRAC 2202 rev2".
Missed approaches are now automatically inserted into the flight plan.
However, this is very time consuming as it is inserted just before the runway and must be deleted each time.
Is it possible for the team to resolve this?

Aircraft Version
 Development

Build info
{
  "built": "2022-03-04T15:37:32+00:00",
  "ref": "master",
  "sha": "0499baea46e2f93b2d5a85b46b96dc341fbb051d",
  "actor": "Benjozork",
  "event_name": "manual",
  "pretty_release_name": "master:0499baea"
}

Route
RJOO/32L ASUKA4 SHTLE Y71 XAC XAC2C RJTT/CREAM. ILS 34L-Z
 
URAGA and UTIBO are used in the missed approach.

![kk2](https://user-images.githubusercontent.com/75520968/156861065-ce52cecd-044d-417c-b457-246071bf0b41.jpg)
![kk](https://user-images.githubusercontent.com/75520968/156861068-552af1c3-87d5-4524-8d59-5044e55b96cc.jpg)
![kk3](https://user-images.githubusercontent.com/75520968/156861214-ed66d511-0031-4d62-9d37-5b495b023498.jpg)

**Comment by tracernz** [comment]
We are ignoring the missed approach legs on our side, so this would be a navigraph error if the missed approach legs appear in the final approach segment.

**Reply by kiokio0987** [reply]
Thank you for the explanation.
I will try to manually remove it until it is resolved.
I will report it to the navigraph forum.

I don't know which will solve the issue, but I hope it will be for the better.

**Reply by tracernz** [reply]
So I just tried loading this approach myself and I didn't get this problem.. Do you have an add-on scenery for this airport at all?

**Reply by kiokio0987** [reply]
I have tried to make the contents of the Community folder only navigraph and A32NX.
However, a missed approach is inserted.
Removing navigraph solves the issue.

EDIT:
Since mods such as the G1000NXi are also affected, the
I'm starting to feel like this is a NAVIGRAPH issue.

**Reply by tracernz** [reply]
Issue confirmed on navi or MS side... we have a work-around inbound https://github.com/flybywiresim/a32nx/pull/6855

**Reply by kiokio0987** [reply]
hmm... If MS and navi don't fix it, I'll have to report it to the add-on developers other than A32NX...

**Reply by kiokio0987** [reply]
The problem is solved in 2202rev3.
Thank you very much for your support.
